<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>374</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>171</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="Operand1">
      <property name="text">
       <string>Операнд 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineOp1"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>90</y>
     <width>158</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="Operand2">
      <property name="text">
       <string>Операнд 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineOp2"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>90</y>
     <width>166</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="Result">
      <property name="text">
       <string>Результат</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineRes"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>195</width>
     <height>200</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QRadioButton" name="btn_Sum">
      <property name="text">
       <string>Сложение</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="btn_Vich">
      <property name="text">
       <string>Вычитание</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="btn_Umn">
      <property name="text">
       <string>Умножение</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="btn_Del">
      <property name="text">
       <string>Деление</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>501</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnResult">
      <property name="text">
       <string>Посчитать</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPush">
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
